runner_test.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+import unittest
+from app import app
+
+
+class LaunchingTest(unittest.TestCase):
+
+    def setUp(self):
+        self.app = app.test_client()
+        self.app.testing = True
+
+    def test_valid_url(self):
+        result = self.app.get('/')
+        self.assertEqual(result.status_code, 200)
+
+    def test_invalid_url_page_not_found(self):
+        result = self.app.get('/a/a')
+        assert 'Error 404' in str(result.data)
```

Comment:
please check the status code of the response, at least additionally. that's the important patr; the site's content is not that important